TeachingRecordSystem/TeachingRecordSystem.sln

```diff
@@ -31,7 +31,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TeachingRecordSystem.Core.T
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TeachingRecordSystem.SupportUi.EndToEndTests", "tests\TeachingRecordSystem.SupportUi.EndToEndTests\TeachingRecordSystem.SupportUi.EndToEndTests.csproj", "{EB46905A-F7B4-4BDC-B376-6091264D6303}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TeachingRecordSystem.SupportUi.Tests", "tests\TeachingRecordSystem.SupportUi.Tests\TeachingRecordSystem.SupportUi.Tests.csproj", "{407EB5A5-8B4F-4F7D-80BA-E39AB6AD013C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TeachingRecordSystem.SupportUi.Tests", "tests\TeachingRecordSystem.SupportUi.Tests\TeachingRecordSystem.SupportUi.Tests.csproj", "{407EB5A5-8B4F-4F7D-80BA-E39AB6AD013C}"
```

Comment:
Nit: can we undo this change?

Reply:
I keep doing it but not sure why it keeps re-applying it!

Reply:
done